src/CommunityToolkit/Xamarin.CommunityToolkit/Converters/VariableMultiValueConverter.shared.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Globalization;
+using System.Linq;
+using Xamarin.CommunityToolkit.Extensions;
+using Xamarin.CommunityToolkit.Helpers;
+using Xamarin.Forms;
+using Xamarin.Forms.Internals;
+
+namespace Xamarin.CommunityToolkit.Converters
+{
+	public class VariableMultiValueConverter : MultiValueConverterExtension, IMultiValueConverter
+	{
+		public MultiBindingCondition ConditionType { get; set; }
+
+		public int Count { get; set; }
+
+		public object Convert(object[] values, Type targetType, object parameter, CultureInfo culture)
+		{
+			if (values == null)
+				return false;
+
+			var boolValues = values.OfType<bool>();
+
+			if (boolValues.Count() != values.Count())
+				return false;
+
+			var count = boolValues.Where(v => v).Count();
```

Comment:
Do we need this Where? If don't we can calculate the `Count` for `boolValues` just one time.

Reply:
@pictos do you mean ```var count = boolValues.Count(v => v);``` ?

Reply:
Yes